src/components/material/MaterialView.tsx

```diff
@@ -101,7 +101,7 @@ export class MaterialView extends Component<MaterialViewProps, MaterialViewState
                         tabIndex={1}
                         isMulti={false}
                         isClearable={false}
-                        options={sortedProviders}
+                        options={sortedProviders || []}
```

Comment:
can we do it @ line no 89 rather than doing it here?

Reply:
Done